automation.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This should also be in a `ci/` directory

Also, if you found this file somewhere, you should cite from where

Reply:
I created it myself using the documentation for speculos. Should I add that?